BundID/Views/Setup/SetupPINLetterIntro.swift

```diff
@@ -2,8 +2,8 @@ import SwiftUI
 import ComposableArchitecture
 
 enum SetupTransportPINIntroAction: Equatable {
-    case chooseHasPINLetter
-    case chooseHasNoPINLetter
+    case availablePINLetter
```

Comment:
Currently all user initiated actions (at least I hope all) are verbs. I agree that `chooseHas` as a prefix here in both cases is a bit awkward, but I think having this naming convention helps overall. What do you think?